cluster/saltbase/salt/cni/init.sls

```diff
@@ -19,9 +19,9 @@ cni-tar:
     - user: root
     - name: /opt/cni
     - makedirs: True
-    - source: https://storage.googleapis.com/kubernetes-release/network-plugins/cni-9d5e6e60e79491207834ae8439e80c943db65a69.tar.gz
+    - source: https://storage.googleapis.com/kubernetes-release/network-plugins/cni-a1802e47123688f6e84d8b297ce5a95ce724ac27.tar.gz
     - tar_options: v
-    - source_hash: md5=1ac2f8c72244f0188b8ba3d6052f9fc1
+    - source_hash: md5=6a8c115d040e6a24695bdcc9d8f6622d
```

Comment:
If we're not going to fork into third_party and the upstream pr isn't going to make it in time, please add a RELEASES file to the same bucket with exactly what this release contains. Also add a comment here and in the gci script that this is not just the stock CNI binary, with a pointer to the RELEASES file.